test/integration/clone_project_test.rb

```diff
@@ -35,7 +37,7 @@ def set_cell(ii, val)
     click_on 'Manual Entry'
     fill_in 'Dataset Name', with: 'I Like Clones'
     find('#edit_table_add_2').click
```

Comment:
Wondering if what's the difference between edit_table_add_1 and 2 is. Similar to my comment on edit_table_save_1 and 2. If this is the best way to name this variable, I'm fine with that as long as we specify the difference in comments.